openmetadata-ui/src/main/resources/ui/src/pages/DatasetDetailsPage/DatasetDetailsPage.component.tsx

```diff
@@ -773,7 +774,7 @@ const DatasetDetailsPage: FunctionComponent = () => {
       if (response) {
         const { version, owner: ownerValue, tags } = response;
         setCurrentVersion(version?.toString());
-        setTableDetails(response);
+        setTableDetails((previous) => ({ ...response, ...previous }));
```

Comment:
@Ashish8689 , Did we verify that we are getting the expected result wherever we are using the `tableDetails` data?

Reply:
This will override the changes, please do not check in such things

Reply:
@Sachin-chaurasiya  yes, we have verified it.